src/client/components/CountryMultiSelect/CountryMultiSelect.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Country } from 'meta/area'
+
+import Select, { SelectProps } from 'client/components/Inputs/Select'
+
+import { useCountriesByRegionOptions } from './hooks/useCountriesByRegionOptions'
+
+type Props = Omit<SelectProps, 'options'> & {
+  countries: Array<Country>
+  excludedRegions: Array<string>
+}
+
+const CountryMultiSelect: React.FC<Props> = (props: Props) => {
+  const { countries, excludedRegions, ...selectProps } = props
+  const { multiLabelSummaryKey, placeholder } = selectProps
+
+  const { t } = useTranslation()
+
+  const optionGroups = useCountriesByRegionOptions({ countries, excludedRegions })
+
+  return (
+    <Select
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...selectProps}
+      multiLabelSummaryKey={multiLabelSummaryKey ?? 'admin.country'}
+      options={optionGroups}
+      placeholder={placeholder ?? t('common.countries')}
+    />
+  )
+}
+
+export default CountryMultiSelect
```

Comment:
To be able to use this generic component as a filter I see two options:
1. Adding a new filter type "custom" that allows passing your own filter component which handles the state changes. The drawback of it is that for example in this case, the logic to handle the tooltip would need to be repeated, as that logic is currently part of the filter -> MultiSelect.

2. Keep using MultiSelect and allow passing a custom component to it. I tried this option but passing the props `{
  countries: Array<Country>
  excludedRegions: Array<string>
}` to the CountryMultiSelect made the ts typing too convoluted. 

Any ideas for a clean way to integrate this to the table paginated filters? @sorja @minotogna